src/views/plans/list/PlansListPage.tsx

```diff
@@ -0,0 +1,57 @@
+import type { FC } from 'react';
+import { loadUserSettings } from 'src/components/common/Page/userSettings';
+import StandardPage from 'src/components/page/StandardPage';
+import useGetDeleteAndEditAccessReview from 'src/modules/Providers/hooks/useGetDeleteAndEditAccessReview';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { PlanModel, PlanModelGroupVersionKind, type V1beta1Plan } from '@kubev2v/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+import PlansAddButton from '../../../modules/Plans/components/PlansAddButton';
+import PlansEmptyState from '../../../modules/Plans/components/PlansEmptyState';
+
+import PlanRow from './components/PlanRow/PlanRow';
+import { planFields } from './utils/planFields';
+
+import './PlansListPage.style.css';
+
+type PlansListPageProps = {
+  namespace: string;
+};
+
+const PlansListPage: FC<PlansListPageProps> = ({ namespace }) => {
+  const { t } = useForkliftTranslation();
+
+  const userSettings = loadUserSettings({ pageId: 'Plans' });
+
+  const [plans, plansLoaded, plansLoadError] = useK8sWatchResource<V1beta1Plan[]>({
+    groupVersionKind: PlanModelGroupVersionKind,
+    isList: true,
+    namespace,
+    namespaced: true,
+  });
+
+  const { canCreate } = useGetDeleteAndEditAccessReview({
+    model: PlanModel,
+    namespace,
+  });
+
+  return (
+    <StandardPage
+      data-testid="network-maps-list"
+      {...(canCreate && {
+        addButton: <PlansAddButton dataTestId="add-network-map-button" namespace={namespace} />,
```

Comment:
hide the button? no disable it? move canCreate as prop isDisabled to PlansAddButton ?